docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  mysql:
+    image: mysql:latest
```

Comment:
Using `mysql:latest` is not recommended for production environments as it can lead to unexpected changes. Consider specifying a specific version of the MySQL image to ensure consistency.